docs/source/examples.rst

```diff
@@ -437,15 +446,18 @@ This type of connection does not automatically respond to a "ping" with a "pong"
   ws.connect("ws://echo.websocket.org")
   ws.ping()
   ws.ping("This is an optional ping payload")
-  ws.pong()
   ws.close()
 
 **WebSocketApp ping/pong example**
 
 This example, and ``run_forever()`` in general, is better for long-lived connections.
-If a server needs a regular ping to keep the connection alive, this is probably
-the option you will want to use. The ``run_forever()`` function will automatically
-send a "pong" when it receives a "ping", per the specification.
+
+In this example, if a ping is received, and pong is sent as response, and then the
+client is notified, via ``on_ping()``.
+
+Further, a ping is transmitted every 60 seconds. If a pong is received, the client
+is notified, via ``on_pong()``. If no pong is received, within 10 seconds,
+``run_forever()`` will exit with an ``WebSocketTimeoutException``.
```

Comment:
nitpick: should be `a WebSocketTimeoutException`, not `an WebSocketTimeoutException`

Reply:
Well spotted. Fixed.